canonical-web-and-design/maas-ui
# feat(ui): added the machine details route

## Done

- Added routing for the machine details and added basic page structure.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the machine list and click on a machine.
- You should navigate to the machine details route and see the fqdn.

## Fixes

Fixes: canonical-web-and-design/MAAS-squad#2101
